content/blog/2025/10/2025-10-29-conference-wrap-up.md

```diff
@@ -0,0 +1,70 @@
+
+The first success was of course in attendance. With more than 250 attendees on-site and hundreds of attendees online, the Conference was _attractive_ to many. This is particularly remarkable because the majority of our audience is in Germany, and this year's Conference was held in France. We want to thank our community for being so dedicated to Matrix and for following us here!
+
+{{ figure(
+    img="/blog/img/conf25-keynote.webp"
+    caption=""
+    alt="A picture of Amandine, one of the Guardians of The Matrix.org Foundation, standing in front of a large crowd in a packed room. Her slides read \"The Foundation Update\"."
+) }}
+
+<!-- more -->
+
+The Conference started by a Barcamp/Unconference: it's an informal workshop day where people join and share the topics they want to talk and hear about. In the evening we all gathered for a Welcome Party at bar Le Météor. Rocket.Chat and Element sponsored the evening, so attendees only had to come and grab food and drinks. We're grateful to both Rocket.Chat and Element for offering us this great night of socializing!
+
+{{ figure(
+    img="/blog/img/conf25-yan-barcamp.webp"
+    caption=""
+    alt="A photo of a white man wearing a hat and a purple hoodie. He holds a microphone and speaks in the middle of a crowd where many people are busy on their computers and talking to each other."
+) }}
+
+The second and third days were the heart of the conference. The global show-up rate was high! Indeed, 96% of people who bought a ticket showed up. We asked people to optionally add their country of origin, and for those who replied 20 countries were represented. We got 68 speakers, including 2 remote sessions and 1 hybrid session.
+
+On that note, _more than a day's worth of video_ were recorded by the excellent Audio/Video team. The [Chaos Computer Club Video Operation Center](https://c3voc.de/) indeed brought their audio and video equipment and expertise, and have set up a professional recording and streaming set up. They trained volunteers to use their equipment and home-developed software. Volunteers found it easy to operate the camera and audio/video mixers.
+
+TODO: insert picture of Dominik operating the video mixer and the picture of Tulir operating a camera
+
+Amazingly, talks were not only live-streamed, they were then available on [our YouTube Channel](https://www.youtube.com/watch?v=Hvdlk3kz4wY&list=PLl5dnxRMP1hUgnYEbpEsEEhIqY_KlO3NG) and on the Chaos Computer Club's [own streaming platform](https://media.ccc.de/c/matrix-conf-2025) _within minutes_ after the speaker stopped talking. Even more impressive? The VOC team members are volunteers.
+
+{{ figure(
+    img="/blog/img/conf25-av.webp)"
+    caption=""
+    alt="A photo of a woman wearing a purple hoodie with the logo of The Matrix Conference embroidered prominently. She is looking at another volunteer, smiling. The other volunteer is operating video mixing software on a laptop."
+) }}
```

Comment:
Why doesn't the picture load for me?
<img width="1067" height="514" alt="image" src="https://github.com/user-attachments/assets/25ab86e8-d28b-4c49-aea7-6fb04a58b081" />